src/Tablebot/Plugins/Quote.hs

```diff
@@ -33,10 +39,28 @@ share
 Quote
     quote String
     author String
+    msgId Int Maybe
+    cnlId Int Maybe
+    time UTCTime
     deriving Show
 |]
 
--- | Our quote command, which combines @addQuote@ and @showQuote@.
+-- | @quoteReactionAdd@ creates an event for when a reaction is added to a message.
+-- If the reaction is :speech_balloon:, the message is added as a quote
+quoteReactionAdd :: ReactionAdd
+quoteReactionAdd = ReactionAdd qra
+  where
+    qra ri
+      | emojiName (reactionEmoji ri) == "\x1F4AC" = do
+        m <- m'
```

Comment:
`m'` is not a great name here, especially as it doesn't have the same type as `m` (`m` is the result of a computation, `m'` is a computation)